tests/toolchain/build-pass/no_abigen_test.json

```diff
@@ -0,0 +1,10 @@
+{
+   "tests" : [
+     {
+       "compile_flags": ["--no-abigen"],
+       "expected" : {
+         "exit-code": 0,
```

Comment:
Remove `,` will fix it.

Reply:
Yes, only one line inside {} should not has ','